test-harness/src/test/groovy/com/netflix/conductor/test/integration/DecisionTaskSpec.groovy

```diff
@@ -183,6 +183,9 @@ class DecisionTaskSpec extends AbstractSpecification {
         when: "the task 'integration_task_20' is polled and completed"
         def polledAndCompletedTask20Try1 = workflowTestUtil.pollAndCompleteTask('integration_task_20', 'task1.integration.worker')
 
+        and: "the workflow is evaluated"
+        sweep(workflowInstanceId)
```

Comment:
why this changes in tests? same comment for the others?

Reply:
As part of the changes introduced, the workflow will be evaluated lazily for a join task, so we simulate the lazy evaluation by calling a sweep here.